src/js/components/NotFound.jsx

```diff
@@ -0,0 +1,25 @@
+// src/components/NotFound.jsx
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default class NotFound extends React.Component {
+
```

Comment:
express as a functional component. refer to the comment about functional components in file `src/js/components/Footer.jsx`